app/src/main/res/layout-land/profile_chooser_fragment.xml

```diff
@@ -0,0 +1,115 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:app="http://schemas.android.com/apk/res-auto">
+
+  <data>
+
+    <variable
+      name="viewModel"
+      type="org.oppia.app.profile.ProfileChooserViewModel" />
+  </data>
+
+  <androidx.constraintlayout.widget.ConstraintLayout
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+    <androidx.core.widget.NestedScrollView
+      android:layout_width="match_parent"
+      android:layout_height="match_parent"
+      android:background="@color/profileChooserBackground"
```

Comment:
Color should be named ``profile_chooser_background`` for consistency.

Reply:
@BenHenning For consistency we introduced camel-casing, unless its shades of different colors. PTAL on https://github.com/oppia/oppia-android/blob/develop/app/src/main/res/values/colors.xml

Reply:
Ack, similarly let's keep this for now. Later, we should make everything consistent.